_posts/2017-10-12-Rust-1.21.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: post
+title: "Announcing Rust 1.21"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.21.0. Rust
+is a systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed, getting Rust 1.21 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.21.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1210-2017-10-12
+
+### What's in 1.21.0 stable
+
+This release contains some very minor, but nice-to-have features, as well as
+some new documentation.
+
+First up, a small change to literals. Consider code like this:
+
+```rust
+let x = &5;
+```
+
+In Rust, this code is synonymous with:
+
+```rust
+let _x = 5;
+let x = &x;
```

Comment:
This code doesn't compile.....

```
error[E0425]: cannot find value `x` in this scope
 --> src/main.rs:3:10
  |
3 | let x = &x;
  |          ^ did you mean `_x`?
```

I think it should be:

```
let x = 5;
let x = &x;
```

?

Reply:
`&_x` was what I intended; it depends if you think that shadowing obscures or helps the example. oops!